index.html

```diff
@@ -4,15 +4,189 @@
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0"
+    />
     <link rel="stylesheet" href="style.css" />
-    <title>Document</title>
+    <title>Home</title>
   </head>
   <body>
-    <header>
-      <h1 class="title">this is the header</h1>
+    <header class="toolbar">
+      <menu class="menu_items">
+        <p class="close_menu_icon">X</p>
+        <ul>
+          <li><a href="./index.html" tabindex="1">Home</a></li>
+          <li><a href="./about.html" tabindex="2">About</a></li>
+          <li><a href="#main_program" tabindex="3">Program</a></li>
+          <li><a href="#speakers" tabindex="4">Speakers</a></li>
+          <li><a href="#sponsors" tabindex="5"> Sponsors</a></li>
+        </ul>
+      </menu>
+      <nav class="menu_icon">
+        <i class="fa-solid fa-bars" id="menu_icon"></i>
+      </nav>
+      <div class="nav_items">
+        <div class="toolbar_top">
+          <ul class="socials">
+            <li class="icons icons1">
+              <a href="https://twitter.com/EzzyWealth"
+                ><i class="fa-brands fa-twitter"></i
+              ></a>
+            </li>
+            <li class="icons icons2">
+              <a href="https://www.linkedin.com/in/ezekiel-udiomuno-b14539150/">
+                <i class="fa-brands fa-linkedin-in"></i
+              ></a>
+            </li>
+
+            <li class="icons icons4">
+              <a href="https://github.com/Ezzywealth"
+                ><i class="fa-brands fa-github"></i
+              ></a>
+            </li>
+          </ul>
+          <ul>
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+          </ul>
+        </div>
+        <div class="toolbar_bottom">
+          <a href="./index.html"
+            ><img src="./images/javascript-39424.png" alt="logo" class="logo"
+          /></a>
+
+          <nav>
+            <a href="#" tabindex="1">Home</a>
+            <a href="./about.html" tabindex="2">About</a>
+            <a href="#main_program" tabindex="3">Program</a>
+            <a href="#speakers" tabindex="4">Speakers</a>
+            <a href="#sponsors" tabindex="5"> Sponsors</a>
+            <h3 class="js_camp">JS Campaign</h3>
+          </nav>
+        </div>
+      </div>
     </header>
     <main>
-      <section>this is the headline section</section>
+      <section class="headline_section">
+        <article class="headline_item">
+          <h1 class="headline_header">
+            #1 Javascript <br />
+            Conference On the Planet
+          </h1>
```

Comment:
- [ ] To match the template design. I think it would be better if you update some styling of the page title such as: `upercase` text transformation, increase  a little bit the `font size`, `line height`,... The important is to make sure its styling is so close as  much as possible the template design